bundles/org.openhab.binding.ferroamp/src/main/java/org/openhab/binding/ferroamp/internal/FerroampBindingConstants.java

```diff
@@ -0,0 +1,207 @@
+    public static final String CHANNEL_SOH = "soh";
+    public static final String CHANNEL_POWERBATTERY = "power-battery";
+    public static final String CHANNEL_TOTALCAPACITYBATTERIES = "total-capacity-batteries";
+
+    // List of SSO Channel ids
+    public static final String CHANNEL_S1ID = "s1-id";
+    public static final String CHANNEL_S1PVVOLTAGE = "s1-pv-voltage";
+    public static final String CHANNEL_S1PVCURRENT = "s1-pv-current";
+    public static final String CHANNEL_S1TOTALSOLARENERGY = "s1-total-solar-energy";
+    public static final String CHANNEL_S1RELAYSTATUS = "s1-relay-status";
+    public static final String CHANNEL_S1TEMPERATURE = "s1-temperature";
+    public static final String CHANNEL_S1FAULTCODE = "s1-fault-code";
+    public static final String CHANNEL_S1DCLINKVOLTAGE = "s1-dc-link-voltage";
+    public static final String CHANNEL_S1TIMESTAMP = "s1-timestamp";
+    public static final String CHANNEL_S2ID = "s2-id";
+    public static final String CHANNEL_S2PVVOLTAGE = "s2-pv-voltage";
+    public static final String CHANNEL_S2PVCURRENT = "s2-pv-current";
+    public static final String CHANNEL_S2TOTALSOLARENERGY = "s2-total-solar-energy";
+    public static final String CHANNEL_S2RELAYSTATUS = "s2-relay-status";
+    public static final String CHANNEL_S2TEMPERATURE = "s2-temperature";
+    public static final String CHANNEL_S2FAULTCODE = "s2-fault-code";
+    public static final String CHANNEL_S2DCLINKVOLTAGE = "s2-dc-link-voltage";
+    public static final String CHANNEL_S2TIMESTAMP = "s2-timestamp";
+    public static final String CHANNEL_S3ID = "s3-id";
+    public static final String CHANNEL_S3PVVOLTAGE = "s3-pv-voltage";
+    public static final String CHANNEL_S3PVCURRENT = "s3-pv-current";
+    public static final String CHANNEL_S3TOTALSOLARENERGY = "s3-total-solar-energy";
+    public static final String CHANNEL_S3RELAYSTATUS = "s3-relay-status";
+    public static final String CHANNEL_S3TEMPERATURE = "s3-temperature";
+    public static final String CHANNEL_S3FAULTCODE = "s3-fault-code";
+    public static final String CHANNEL_S3DCLINKVOLTAGE = "s3-dc-link-voltage";
+    public static final String CHANNEL_S3TIMESTAMP = "s3-timestamp";
+    public static final String CHANNEL_S4ID = "s4-id";
+    public static final String CHANNEL_S4PVVOLTAGE = "s4-pv-voltage";
+    public static final String CHANNEL_S4PVCURRENT = "s4-pv-current";
+    public static final String CHANNEL_S4TOTALSOLARENERGY = "s4-total-solar-energy";
+    public static final String CHANNEL_S4RELAYSTATUS = "s4-relay-status";
+    public static final String CHANNEL_S4TEMPERATURE = "s4-temperature";
+    public static final String CHANNEL_S4FAULTCODE = "s4-fault-code";
+    public static final String CHANNEL_S4DCLINKVOLTAGE = "s4-dc-link-voltage";
+    public static final String CHANNEL_S4TIMESTAMP = "s4-timestamp";
+
+    // List of ESO Channel ids
+    public static final String CHANNEL_ESOID = "eso-id";
```

Comment:
Applies to _all_ channels, remove the device/thing prefix.

For example
`eso-id`, `esm-id`, `s2-id` etc etc they can also be just `id`

Reply:
Shouldn't it be like:
public static final String CHANNEL_S1ID = "s1-id";

and
public static final String CHANNEL_S2ID = "s2-id";

and so on
regarded to the channel id in thing-types.xml

<channel id="s1-id" typeId="string">
				<label>S1 ID</label>
			</channel>

and 
<channel id="s2-id" typeId="string">
				<label>S2 ID</label>
			</channel>